### PR TITLE
Adds pronoun pins

### PR DESCRIPTION
**Pull request by Llywelwyn**
Adds a selection of pronouns pins into the game. 

I've seen a lot of interest in this but it's usually been dismissed with the fact that you can just rename one of the existing pins, which whilst true, the hope here is that it will normalise their use, so players will feel less uncomfortable about using one on their characters, which was something expressed by more than one player when I asked around before starting on this.

Plus, to differentiate it from the regular pin this has its own unique sprite (modified from bay) which can be recoloured.

image
image

### CODE REVIEW

File: code/modules/clothing/under/accessories/accessory.dm [hunk 1]
```diff
@@ -1269,3 +1270,49 @@
...
+	worn_overlay = "over"
+	drop_sound = 'sound/items/drop/card.ogg'
+	pickup_sound = 'sound/items/pickup/card.ogg'
+	contained_sprite = TRUE
+	build_from_parts = TRUE
+	overlay_in_inventory = FALSE
+
+/obj/item/clothing/accessory/pronoun/hehim
+	name = "he/him pronouns pin"
+	desc = "A pin denoting the wearer's pronouns: any/all."
```
Comment (Greenjoe12345) [inline code/modules/clothing/under/accessories/accessory.dm]: wrong desc here

File: code/modules/clothing/under/accessories/accessory.dm [hunk 2]
```diff
@@ -1269,3 +1270,49 @@
...
+
+/obj/item/clothing/accessory/pronoun/hehim
```
Comment (Greenjoe12345) [inline code/modules/clothing/under/accessories/accessory.dm]: you've got the he/him pin twice

Comment (Llywelwyn) [reply code/modules/clothing/under/accessories/accessory.dm]: Just noticed this myself, oops. Fixed these with https://github.com/Aurorastation/Aurora.3/pull/16446/commits/9a6aac672b40e32102b6849d156e00dae857c1e9

**Comment by Llywelwyn** [comment]
!review

**Comment by alsoandanswer** [comment]
Shouldn't this be under the regular ID tag, if not already?

**Comment by Llywelwyn** [comment]
> Shouldn't this be under the regular ID tag, if not already? We already have pronoun selection in the character creation menu.

Pronouns aren't on IDs currently. But even if it were, we already have precedent for plenty of items to make those things clearer so that someone doesn't have to show off their ID. Blood type is on the ID, but we have blood patches to make that clearer; employer is on the ID, but we have badges for that too.

**Comment by alsoandanswer** [comment]
From a purely code perspective, the thing is, blood type and corporation are very few and defined. However, pronouns are pretty personal and are very varied. The custom name and description system was made specifically to combat loadout bloat such as this.

Aurora is already a pretty LGBT friendly server compared to others (although that's not a very high bar to cross) 

I understand the sentiment of supporting people, but I have an *overwhelming* premonition the real people this is going to spur on are shitters spawning with nametags like attack helicopter, while the actual LGBT members of this community will largely ignore these items and use custom tags anyway.

**Comment by CyberSpyX** [comment]
The name of this PR is misleading. Please rename it to something related to the pronoun pins.

**Comment by Llywelwyn** [comment]
> I understand the sentiment of supporting people, but I have an _overwhelming_ premonition the real people this is going to spur on are shitters spawning with nametags like attack helicopter, while the actual LGBT members of this community will largely ignore these items and use custom tags anyway.

If this is how the maintainers feel about it, I'm fine with closing, but I'd encourage asking around or polling or something to see how people do actually feel about it overall.

I was 99% sure this wouldn't be approved before making this PR in the first place, but when deciding if I'd do it or not I asked a few players (who I knew to play nonbinary characters in the past), and pretty much overwhelmingly the idea was liked, and most people said it'd make them feel more comfortable/less like a snowflake for custom-making a pin. I didn't get any negative feedback from anyone I asked.

**Review by Arrow768 — APPROVED** [review]
You need to merge master into your PR to ensure that the current UTs are executed.

In the future make sure that the title of your PR properly reflects what it does when that PR is created.

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**